libuserhelper.cpp

```diff
@@ -176,7 +181,9 @@ uint LibUserHelper::addUser(const QString &user, const QString& name, uint uid,
     uint rv = 0;
     struct lu_ent *ent_user = lu_ent_new();
     if (lu_user_default(context, user.toUtf8(), false, ent_user)) {
-        QString fixedName(name);
+        QString nameUuid(name);
+        nameUuid += ',';
+        nameUuid += QUuid::createUuid().toString();
```

Comment:
This is not necessarily an error but to me it feels like this uuid creation should be inside SailfishUserManager class and the value should be just passed to LibUserHelper that would then just save it.

BTW, at the moment user can change their gecos information with chfn command from util-linux and this allows user to change this uuid as well (now it's saved as office information, option -o).

Reply:
Just thinking to avoid this problem we should consider either:
- removing chfn
- altering chfn so that we can store extra fields to the end that can't be changed with chfn

Reply:
Ehm. Sorry, it's actually prevented already on Sailfish. You may ignore this.
`chfn: /etc/login.defs: CHFN_RESTRICT does not allow any changes`